test/gtest/Makefile.am

```diff
@@ -27,6 +28,12 @@ export UCX_HANDLE_ERRORS
 export UCX_LOG_LEVEL
 export UCX_LOG_PRINT_ENABLE
 
+if HAVE_ASAN
+	ENV = \
+		LSAN_OPTIONS=suppressions=$(top_srcdir)/contrib/lsan.supp \
+		ASAN_OPTIONS=protect_shadow_gap=0
```

Comment:
why not set it as other env vars, for example UCX_LOG_PRINT_ENABLE in line 18

Reply:
Done.